src/common/lib/hid_interface.c

```
@@ -159,397 +161,449 @@ static inline bool hid_send_report(uint8_t instance, uint8_t report_id, void con
-      hid_send_report(ITF_NUM_KEYBOARD, REPORT_ID_KEYBOARD, &keyboard_report,
-                      sizeof(keyboard_report));
-    }
-  } else if (IS_CONSUMER(code)) {
-    uint16_t usage;
-    if (make) {
-      usage = CODE_TO_CONSUMER(code);
-    } else {
-      usage = 0;
+        // Shift-override: temporarily remove shift modifiers if suppression requested
+        uint8_t saved_modifiers = 0;
+        if (suppress_shift) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)